### PR TITLE
Final fixing done

### PR DESCRIPTION
**Pull request by Priyanshu0713**

### CODE REVIEW

File: src/layouts/Container/index.js [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+
+});
+
+function Containerbox() {
+  const classes = useStyles();
+
+  return (
+    <div style={{backgroundColor:"white"}}><><><div>
+      <Head>
+        <title>Home-Hashtag404</title>
```
Comment (Rishuhashtag) [inline src/layouts/Container/index.js:152]: please use heading tag

File: src/layouts/Conversation/index.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  return (
+    <div style={{backgroundColor:"white"}}>
+      <Head>
+        <title>Home-Hashtag404</title>
+        <meta
+          name="description"
+          content="Meta description for the Home page THIS IS AS HOME PAGE" />
+        <meta name="keywords" content="test1, test2, ssr, nextjs" />
+      </Head>
+ {/* >>>>>>>>>>>>>>>     Interested       >>>>>>>>>>>>>>>>>>>>>>>>>>>> */}
```
Comment (Rishuhashtag) [inline src/layouts/Conversation/index.js:52]: please use /*your comment */

Comment (Priyanshu0713) [reply src/layouts/Conversation/index.js:52]: it did't work like thar
i used /*...*/ this but it did't work thats why i used {/*...*/} this

File: src/layouts/Container/index.js (hunk 1 above)
Comment (Priyanshu0713) [reply src/layouts/Container/index.js:152]: did't get that point .can you show me how?

File: src/layouts/Navbar/Containerbox.js [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+    marginRight: 20,
+  },
+  img_mentor: {
+    width: "30%",
+    height: "10%",
+    borderRadius: "50%",
+    objectFit: "cover",
+    marginBottom: -10,
+  },
+
```
Comment (Rishuhashtag) [inline src/layouts/Navbar/Containerbox.js:98]: remove this space

File: src/layouts/Navbar/Containerbox.js [hunk 4]
```diff
@@ -0,0 +1,282 @@
...
+function Containerbox() {
+  const classes = useStyles();
+
+  return (
+    <div style={{ backgroundColor: "white" }}>
+      <>
+        <>
+          <div>
+            <Head>
+              <title>Home-Hashtag404</title>
```
Comment (Rishuhashtag) [inline src/layouts/Navbar/Containerbox.js:149]: Use h1-h6 tag for heading

File: src/layouts/Navbar/Containerbox.js [hunk 5]
```diff
@@ -0,0 +1,282 @@
...
+
+function Containerbox() {
+  const classes = useStyles();
+
+  return (
+    <div style={{ backgroundColor: "white" }}>
+      <>
+        <>
+          <div>
+            <Head>
```
Comment (Rishuhashtag) [inline src/layouts/Navbar/Containerbox.js:148]: pls remove <head> & </head> . we are using dv container already.

File: src/layouts/Navbar/Containerbox.js [hunk 6]
```diff
@@ -0,0 +1,282 @@
...
+              <title>Home-Hashtag404</title>
+            </Head>
+
+            {/* Container*/}
```
Comment (Rishuhashtag) [inline src/layouts/Navbar/Containerbox.js:152]: Remove this line

File: src/layouts/Navbar/Header.js [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+    backgroundColor: "white",
+    textAlign: "center",
+  },
+  h3: {
+    fontSize: "2.5rem",
+  },
+  h: {
+    fontSize: "1.2rem",
+  },
+
```
Comment (Rishuhashtag) [inline src/layouts/Navbar/Header.js:24]: Remove extra empty row like 24, 10, 13, 46,48, 53, 55, 88

File: src/layouts/Navbar/Navbar.js [hunk 8]
```diff
@@ -0,0 +1,252 @@
...
+        ))}
+      </List>
+    </div>
+  );
+
+  return (
+    <div style={{ backgroundColor: 'white' }}>
+      <div>
+        <Head>
+          <title>Home-Hashtag404</title>
```
Comment (Rishuhashtag) [inline src/layouts/Navbar/Navbar.js:193]: please use h1-h6 tag instead of title 
remove head tag

File: src/layouts/Navbar/Navbar.js [hunk 9]
```diff
@@ -0,0 +1,252 @@
...
+          <meta
```
Comment (Rishuhashtag) [inline src/layouts/Navbar/Navbar.js:194]: remove meta part

**Review by im-nikhilkatiyar — APPROVED** [review]

**Pull request merged**